utils/src/main/kotlin/Utils.kt

```diff
@@ -26,11 +26,19 @@ import java.net.URI
 import java.net.URISyntaxException
 import java.security.Permission
 
+import kotlin.reflect.full.memberProperties
+
 /**
  * The name of the ORT configuration file.
  */
 const val ORT_CONFIG_FILENAME = ".ort.yml"
 
+/**
+ * Return whether the [receiver] (usually an instance of a data class) has any non-null property.
+ */
+inline fun <reified T : Any> anyNonNullProperty(receiver: T) =
```

Comment:
I would prefer `hasNonNullProperty` as the name for this function.

Reply:
And you could also make this an extension function for `T`.

Reply:
I was in fact considering that name, but then thought that the `any` prefix better matches the used `any` function. But I'll probably change it anyway as we both thought about it.